perfkitbenchmarker/linux_virtual_machine.py

```diff
@@ -540,6 +540,34 @@ def BurnCpu(self, burn_cpu_threads=None, burn_cpu_seconds=None):
         time.sleep(end_time - time.time())
       self.RemoteCommand('pkill -9 sysbench')
 
+  def PrepareBackgroundWorkload(self):
+    """ Install packages needed for the background workload """
+    if self.background_cpu_threads:
+      self.Install('sysbench')
+
+  def StartBackgroundWorkload(self):
+    """Starts the blackground workload.
+
+    Args:
+      self: The target vm.
+      background_cpu_threads: Number of threads active in the background.
+    """
+    if self.background_cpu_threads:
+      self.RemoteCommand(
+          'nohup sysbench --num-threads=%s --test=cpu --cpu-max-prime=10000000 '
+          'run 1> /dev/null 2> /dev/null &' % self.background_cpu_threads)
+
+  def StopBackgroundWorkload(self):
+    """Stops the background workload.
+
+    Args:
+      vm: The target vm.
+      background_cpu_threads: Number of threads started.
+        Used to determine whether anything needs to be stopped.
+    """
+    if self.background_cpu_threads:
+      self.RemoteCommand('pkill -9 sysbench')
```

Comment:
Could this be problematic if we're running a sysbench benchmark or if we're also using sysbench for some other purpose?